api_responses/response_dtypes.py

```diff
@@ -0,0 +1,47 @@
+from typing import Dict, List, TypeVar, Generic, Literal, Any, Optional
+
+from pydantic import BaseModel, Field
+
+T = TypeVar("T")
+
+
+class ResModel(BaseModel, Generic[T]):
+    message: str
+    request_id: str
+    data: T
```

Comment:
this wrapper will not be used, the backend api will send the json data object directly